### ISSUE TITLE
Endian

### ISSUE THREAD

**Issue by condret** [radare/radare2#6680]
Can we have support for middle and mixed endian

**Comment by radare** [comment]
Yes. Thats why endian is int and not bool but we need samples to try it out. Pdp?

**Comment by condret** [comment]
e cfg.bigendian

**Comment by radare** [comment]
Yep it will require some more work..

**Comment by trufae** [comment]
This issue should be moved to ideas instead of closing
